src/components/general/header.jsx

```diff
@@ -3,21 +3,32 @@ import Clock from "react-live-clock";
 import checkAuth from "../../scripts/checkAuth";
 import { useAsync } from "react-async";
 import "./header.css";
+import endpointUrl from "../../config";
 
+const handleSignOut = () => {
+  fetch(`${endpointUrl}/signout`, {
+    credentials: "include"
+  }).then(res => (res.status === 302 ? (window.location = "/") : null));
```

Comment:
This doesn't need to be a ternary, because you don't have another `.then()`, so you don't need to return anything. You could just have this:
```js
.then(res => {
    if(res.status === 302) {
        window.location = "/"
    }
})
```